k8s/templates/db-migration-job.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "studio.fullname" . }}-db-migrate
+  labels:
+    tier: backend
+    app: {{ template "studio.fullname" . }}
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  activeDeadlineSeconds: 7200 # 2 hours
+  template:
+    metadata:
+      name: {{ template "studio.fullname" . }}-db-migrate
+    spec:
+      restartPolicy: OnFailure
```

Comment:
What was your reasoning in making these jobs restart on failure?  Would a timeout be considered failure?

Reply:
@micahscopes timeouts will stop it from running again, i think this `OnFailure` means it's for when the image exits with a non-zero exit code, it tries again. If it keeps failing it'll back off from restarting. Once we have `sentry-kubernetes` built in we'll get alerts when we error out like that!